app/models/infrastructure_facades/cloud_facade.rb

```diff
@@ -147,6 +150,22 @@ def get_sm_record_by_id(record_id)
     CloudVmRecord.find_by_id(record_id)
   end
 
+  ##
+  # Returns list of hashes representing distinct types of infrastructure
```

Comment:
@mpaciore - jako, że nie chciałeś zmieniać nazw metody, to proponuję zmienić dokumentację na "...representing distinct configurations of infrastructure"